snuba/datasets/errors_replacer.py

```diff
@@ -281,13 +282,23 @@ def get_projects_query_flags(
                 exclude_groups_key, 0, 0, withscores=True,
             )
 
-    with sentry_sdk.start_span(op="function", description="execute_redis_pipeline"):
+    with sentry_sdk.start_span(
+        op="function", description="execute_redis_pipeline"
+    ) as span:
         results = p.execute()
+        span.set_tag("results", results)
```

Comment:
do you want to be sending all the results to sentry? what would this give you?

Reply:
Yeah I guess I don't need to, I'm tagging the processed results a few lines down anyway.